engine/src/main/java/org/terasology/logic/players/PlayerFactory.java

```diff
@@ -134,15 +134,15 @@ private Vector3f findSpawnPos(Vector3f targetPos, float entityHeight) {
         int targetBlockY = TeraMath.floorToInt(targetPos.y);
         int targetBlockZ = TeraMath.floorToInt(targetPos.z);
         Vector2i center = new Vector2i(targetBlockX, targetBlockZ);
-        for (BaseVector2i pos : SpiralIterable.clockwise(center).maxRadius(3).scale(2).build()) {
+        for (BaseVector2i pos : SpiralIterable.clockwise(center).maxRadius(250).scale(2).build()) {
 
             Vector3i testPos = new Vector3i(pos.getX(), targetBlockY, pos.getY());
             Vector3i spawnPos = findOpenVerticalPosition(testPos, entityHeight);
             if (spawnPos != null) {
                 return new Vector3f(spawnPos.getX(), spawnPos.getY() + entityHeight, spawnPos.getZ());
             }
         }
-        return null;
+        return targetPos;
```

Comment:
Should we go for `Optional[Vector3i]̀  as return type?

See: https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html